db/migrate/20220527165852_create_foods.rb

```diff
@@ -0,0 +1,12 @@
+class CreateFoods < ActiveRecord::Migration[5.2]
```

Comment:
Similar to the comment above re: models -- do we need to store food data at all?